swagger_server/jobs/pull_topo_changes.py

```diff
@@ -0,0 +1,86 @@
+import argparse
+import json
+import logging
+import os.path
+import sys
+import threading
+import time
+import urllib.request
+
+sys.path.append(
+    os.path.abspath(os.path.join(os.path.dirname(__file__), os.path.pardir))
+)
+
+from messaging.rpc_queue_producer import *
+from utils.db_utils import *
```

Comment:
Yeah.. that's right, but here we need all operations from `db_utils`, read/write/update, and rpc_queue_producer, so I imported with *.

Reply:
I am mostly nitpicking here ("mostly" because the [official recommendation](https://docs.python.org/3/tutorial/modules.html#importing-from-a-package) is to avoid `*` imports in production code), but we just need `DbUtils` and `RpcProducer` though. :-)

Reply:
Sounds good. Updated.